ivadomed/main.py

```diff
@@ -32,8 +32,20 @@ def get_parser():
 
     # MANDATORY ARGUMENTS
     mandatory_args = parser.add_argument_group('MANDATORY ARGUMENTS')
+    command_group = mandatory_args.add_mutually_exclusive_group(required=True)
```

Comment:
Cool command, I didn't know that was a feature!

Reply:
It's actually quite useful!